packages/bundler/src/es6-module-bundler.ts

```diff
@@ -78,7 +78,7 @@ async function prepareBundleModule(
         getOrSetBundleModuleExportName(assignedBundle, sourceUrl, '*');
     bundleSource.body.push(babel.importDeclaration(
         [babel.importNamespaceSpecifier(babel.identifier(starExportName))],
-        babel.stringLiteral(rebasedSourceUrl)));
+        babel.stringLiteral(sourceUrl)));
```

Comment:
There should no longer be an issue of confusion here as `sourceUrl` is always a `resolvedUrl` since it comes from `for (const sourceUrl of [...assignedBundle.bundle.files].sort()) {`

The "rebasedSourceUrl" was a thing back when I was trying to keep them all relative URLs but now I only use resolveUrls until the final product is delivered out of the bundler.

Reply:
I could rename it to resolvedSourceUrl if that clarifies it.